presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -351,17 +351,41 @@ public void insertPartitionedTable(HiveStorageFormat storageFormat)
     public void testDeleteFromUnpartitionedTable()
             throws Exception
     {
-        assertQuery("CREATE TABLE test_delete_unpartitioned (x bigint, y varchar)", "SELECT 1");
-
-        try {
-            queryRunner.execute("DELETE FROM test_delete_unpartitioned");
-            fail("expected exception");
-        }
-        catch (RuntimeException e) {
-            assertEquals(e.getMessage(), "This connector only supports delete where one or more partitions are deleted entirely");
+        for (HiveStorageFormat storageFormat : HiveStorageFormat.values()) {
+            testDeleteFromUnpartitionedTable(storageFormat);
```

Comment:
This test does not need to run for each format since all have the same behavior (just deleting some files)